src/interface.rs

```diff
@@ -79,17 +79,20 @@ impl HWIDevice {
     /// Returns the xpub of a device.
     /// # Arguments
     /// * `path` - The derivation path to derive the key.
+    /// * `expert` - Whether to provide more information or not.
     /// * `chain` - Specify which chain to use.
     pub fn get_xpub(
         &self,
         path: &DerivationPath,
+        expert: bool,
         chain: HWIChain,
     ) -> Result<HWIExtendedPubKey, Error> {
         let output = HWICommand::new()
             .add_flag(HWIFlag::Fingerprint(self.fingerprint))
             .add_chain(chain)
             .add_subcommand(HWISubcommand::GetXpub)
-            .add_path(path, false, false)
+            .add_expert(expert)
```

Comment:
The expert flag needs to be added before the subcommand. That should fix the test failing.